dump/src/util/dump/ExternalizationHelper.java

```diff
@@ -1033,6 +1031,61 @@ public void writeTo( ObjectOutput out ) throws IOException {
 
    static class ClassConfig {
 
+      private static Method findGetter( Method[] methods, String name, Class<?> returnType ) throws NoSuchMethodException {
+         for ( Method m : methods ) {
+            if ( !m.getName().equals(name) ) {
+               continue;
+            }
+            if ( !isGetter(m, returnType) ) {
+               continue;
+            }
+            return m;
+         }
+         throw new NoSuchMethodException();
+      }
+
+      private static Method findSetter( Method[] methods, String name, Class<?> parameterType ) throws NoSuchMethodException {
+         for ( Method m : methods ) {
+            if ( !m.getName().equals(name) ) {
+               continue;
+            }
+            if ( !isSetter(m, parameterType) ) {
+               continue;
+            }
+            return m;
+         }
+         throw new NoSuchMethodException();
+      }
+
+      private static boolean isGetter( Method m, Class<?> returnType ) {
+         if ( m.getParameterTypes().length != 0 ) {
+            return false;
+         }
+         if ( !m.getReturnType().equals(returnType) ) {
+            return false;
+         }
+         return isGetter(m);
+      }
+
+      private static boolean isGetter( Method m ) {
+         return m.getName().startsWith("get") || (m.getName().startsWith("is") && (m.getReturnType() == boolean.class || m.getReturnType() == Boolean.class));
```

Comment:
Isn't `get` / `is` prefix already covered by the name check? It should probably check that there aren't any parameters.